web/src/scheduled_messages.js

```diff
@@ -4,14 +4,10 @@ import render_compose_banner from "../templates/compose_banner/compose_banner.hb
 import render_send_later_modal_options from "../templates/send_later_modal_options.hbs";
 
 import * as channel from "./channel";
-import * as compose from "./compose";
-import * as compose_actions from "./compose_actions";
 import * as compose_banner from "./compose_banner";
-import * as compose_ui from "./compose_ui";
 import {$t} from "./i18n";
 import * as narrow from "./narrow";
 import * as people from "./people";
-import * as popover_menus from "./popover_menus";
```

Comment:
Hmm, I would have hoped that we didn't need `compose` to import `scheduled_messages`, rather than the other way around.

One thing you should always be doing when looking at breaking import cycles is thinking about which direction to cut edges; the tool makes enough random choices that there is rarely a reason to prefer the direction it presents over the opposite, and so about half the time the correct change will be in the other direction.

Reply:
Did you meant `popover_menus` and `scheduled_messages` rather than `compose` and `scheduled_messages` ?

Reply:
But point noted that I should consider both directions when resolving cycles will keep this in mind!